libs/ledger-live-common/src/exchange/swap/index.ts

```diff
@@ -101,6 +101,24 @@ const swapProviders: Record<string, SwapProviderConfig> = {
   },
   ftx,
   ftxus: ftx,
+  oneinch: {
+    ...createExchangeProviderNameAndSignature({
+      name: "1inch",
+      signature: "",
+      publicKey: "",
+    }),
+    needsKYC: false,
+    needsBearerToken: false,
+  },
+  paraswap: {
+    ...createExchangeProviderNameAndSignature({
+      name: "paraswap",
+      signature: "",
+      publicKey: "",
```

Comment:
Not sure if CEX attributes have sense in DEX or it is better have 2 different types (SwapDexProviderConfig, SwapCexProviderConfig, or define attribute as non-required "?") 
@jnicoulaud-ledger, could you help us? public key and signature exists or could exist in CEX context?

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2123/commits/c8822e29ecec8e1c7a4c8f34a9d212788fd50272